src/communication/network_interface_sr.c

```diff
@@ -7577,6 +7612,9 @@ smmon_get_tran_info (THREAD_ENTRY * thread_p, unsigned int rid, char *request, i
       /* 3) packing information
        *    - packing information with sequence of
        *      server info -> transaction info */
+      buffer = PTR_ALIGN (buffer_a, MAX_ALIGNMENT);
```

Comment:
if `buffer_a` is allocated using `malloc ()`, then it is already aligned using MAX_ALIGNMENT.

Reply:
But buffer_a is allocated using db_private_alloc(), and it is given a chunk from lea_heap.